backend/src/test/java/gov/cdc/usds/simplereport/service/sms/TextMessageStatusServiceTest.java

```diff
@@ -37,10 +36,8 @@
 
 class TextMessageStatusServiceTest extends BaseServiceTest<TextMessageStatusService> {
```

Comment:
Same as the above: `_service` and `_dataFactory` were both provided by the superclass.

https://github.com/CDCgov/prime-simplereport/security/code-scanning/75